script.js

```diff
@@ -28,23 +29,41 @@
     }} */
     const getMmrHistoryResponse = JSON.parse(getMmrHistoryResponseJson);
 
-    let mmrChangeThisStream = 0;
     let winCountThisStream = 0;
     let lossCountThisStream = 0;
-    for (const {date_raw: dateUnixS, mmr_change_to_last_game: mmrChange} of getMmrHistoryResponse.data) {
+    let drawCountThisStream = 0;
+    
+    let latestMatchThisStream = 0;
+    let latestRawEloThisStream = null;
+    let earliestMatchThisStream = 100000000000000;
```

Comment:
I assume you're using this for "date this is at an arbitrary point in the future". You can just use `Number.POSITIVE_INFINITY` here.